.pre-commit-config.yaml

```diff
@@ -0,0 +1,26 @@
+exclude: trulens_explain/*
+repos:
+-   repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v4.6.0
+    hooks:
+      - id: trailing-whitespace
```

Comment:
Did a quick skim and picked some that might be relevant. Happy to add `destroyed-symlinks` since that was an actual issue I had when refactoring. We don't have many .json files though